.github/workflows/e2e-tests-manual.yaml

```diff
@@ -28,6 +28,7 @@ jobs:
         - 'manual-symmetric-key'
         - 'manual-x509'
         - 'dps-x509'
+        - 'dps-symmetric-key'
```

Comment:
For consistency, reorder this to put the symmkey before the x509, just like with manual. Both in the YAMLs and in the script.